src/desktop/lib/webpackPublicPath.ts

```diff
@@ -31,6 +31,9 @@ if (process.env.NODE_ENV === "production") {
     "/collect",
     "/collection",
     "/collections",
+    "/consign",
```

Comment:
It doesn't look like consign was converted...?

Reply:
Ah, foggy head on Friday -- fixed.